com.jianguoyun.Nutstore.json

```diff
@@ -0,0 +1,69 @@
+{
+	"app-id": "com.jianguoyun.Nutstore",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "nutstore",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--share=network",
+		"--filesystem=home"
+	],
+
+	"modules": [
+		{
+			"name": "nutstore",
+			"only-arches": ["x86_64"],
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -D -t /app/bin -m 755 nutstore",
+				"install -D -t /app/bin -m 755 apply_extra",
+				"install -D -t /app/share/applications -m 644 com.jianguoyun.Nutstore.desktop",
+				"install -D -m 644 com.jianguoyun.Nutstore.64x64.png /app/share/icons/hicolor/64x64/apps/com.jianguoyun.Nutstore.png",
+				"install -D -m 644 com.jianguoyun.Nutstore.128x128.png /app/share/icons/hicolor/128x128/apps/com.jianguoyun.Nutstore.png",
+				"install -D -t /app/share/metainfo -m 644 com.jianguoyun.Nutstore.appdata.xml"
+			],
+			"sources": [
+				{
+					"type": "extra-data",
+					"url": "https://www.jianguoyun.com/static/exe/installer/nutstore_linux_dist_x64.tar.gz",
+					"sha256": "338b501a64fa924a0ab158336f2ba076f6bd92aa9ad1070c9c95024a8aef9935",
+					"filename": "nutstore_linux_dist_x64.tar.gz",
+					"size": 59289144
+				},
+				{
+					"type": "script",
+					"dest-filename": "nutstore",
+					"commands": [
+						"python3 /app/extra/bin/nutstore-pydaemon.py"
+					]
+				},
+				{
+					"type": "script",
+					"dest-filename": "apply_extra",
+					"commands": [
+						"tar -xf nutstore_linux_dist_x64.tar.gz",
+						"rm -f nutstore_linux_dist_x64.tar.gz"
+					]
+				},
+				{
+					"type": "file",
+					"path": "com.jianguoyun.Nutstore.64x64.png"
+				},
+				{
+					"type": "file",
+					"path": "com.jianguoyun.Nutstore.128x128.png"
+				},
+				{
+					"type": "file",
+					"path": "com.jianguoyun.Nutstore.desktop"
+				},
+				{
+					"type": "file",
+					"path": "com.jianguoyun.Nutstore.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Well, It seems the documents in docs.flatpak.org should be updated then.